src/it/ISSUE-423/invoker.properties

```diff
@@ -0,0 +1,2 @@
+invoker.goals=clean install license:add-third-party
```

Comment:
I don't think `install` is necessary here.  I added it while debugging an issue.